stable/voyager/templates/cluster-role.yaml

```diff
@@ -0,0 +1,69 @@
+{{ if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "name" . }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+rules:
+- apiGroups:
+  - extensions
+  resources:
+  - thirdpartyresources
+  verbs: ["get", "create", "list"]
+- apiGroups:
+  - voyager.appscode.com
+  resources: ["*"]
+  verbs: ["*"]
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs: ["get", "create", "update"]
+- apiGroups:
+  - extensions
+  resources:
+  - deployments
+  - daemonsets
+  - ingresses
+  verbs: ["*"]
+- apiGroups: [""]
+  resources:
+  - replicationcontrollers
+  - services
+  - endpoints
+  - configmaps
+  verbs: ["*"]
+- apiGroups: [""]
+  resources:
+  - secrets
```

Comment:
Voyager needs the ability to view all secrets on the system? Maybe use a `RoleBinding` for these instead of `ClusterRoleBinding` so this is at least limited to a single namespace?

Reply:
Voyager can issue TLS certs from LE. That process requires ability to create, get, update secrets from any namespace as user chooses.

Reply:
Sure this comment was meaning to ask if there was a mode where voyager was limited to a single namespace. If not this is fine.

Reply:
No, we can't restrict Voyager to a single namespace.